pipeline/go.sh

```diff
@@ -461,18 +463,20 @@ function do_all
 }
 
 ######### VARIABLES #########
+: ${WHOAMI="${USER}"}
```

Comment:
For those not so initiated with Bourne shell scripting: This is essentially “If `WHOAMI` is not set, set it to `"${USER}"`”.